www.foia.gov/request.html

```diff
@@ -2,5 +2,6 @@
 title: Create a request
 layout: default
 navigation: request
+not_legacy: true
```

Comment:
How does someone know if the page should be legacy or not?

Reply:
Originally I had this as a `legacy` identifier in the frontmatter of pages we're keeping as-is, and I think it just morphed too far. Do you think changing it to `use_uswds` or similar makes it clearer?